tests/operations/test_macro_einsum.py

```diff
@@ -0,0 +1,105 @@
+
+import pytest
+
+from myia.lib import InferenceError
+from myia.operations import einsum
+
+from ..common import MA, MB
+from ..multitest import infer, mt, run_debug
+
+
+@run_debug(MA(4, 1)[0])
```

Comment:
`MA(4, 1)[0]` returns a vector of size 1, did you mean `MA(1, 4)[0]`? (Same comment for the tests below)

Reply:
Yes